jaxb-ri/runtime/impl/src/main/java/org/glassfish/jaxb/runtime/v2/runtime/JAXBContextImpl.java

```diff
@@ -716,10 +716,6 @@ public UnmarshallerImpl createUnmarshaller() {
         return new UnmarshallerImpl(this,null);
     }
 
-    public Validator createValidator() {
-        throw new UnsupportedOperationException(Messages.NOT_IMPLEMENTED_IN_2_0.format());
```

Comment:
if this message is no longer used, remove it also from properties file

Reply:
Messages.NOT_IMPLEMENTED_IN_2_0 deleted.